plugins/cairo-lang-macro-attributes/src/lib.rs

```diff
@@ -49,7 +49,7 @@ pub fn attribute_macro(_args: TokenStream, input: TokenStream) -> TokenStream {
 ///
 /// # Safety
 #[proc_macro_attribute]
-pub fn aux_data_collection_callback(_args: TokenStream, input: TokenStream) -> TokenStream {
+pub fn post_process_callback(_args: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
```suggestion
pub fn post_process(_args: TokenStream, input: TokenStream) -> TokenStream {
```

imho `callback` was superfluous